OneCommunityGlobal/HighestGoodNetworkApp
# Ramya Unit Test : UsersProfile/ AssignBadgePopup

# Description
Unit test for src/components/UserProfile/AssignBadgePopup
## Main changes explained:
-  New file
      - **_AssignBadgePopup.test.jsx_** : unit test cases 
      - **_MockAssignTabelRow.jsx_** : mock component
-  Existing file
      - **_AssignBadgePopup_**  : Added  test id to the objects for unit testing 
####  Test cases 
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/89441435/49e8b805-eb85-4b23-8945-54186803d92c)


## How to test:
1. check into current branch
2. Uncomment the following line to  test the test cases in local 
    - jest.mock('axios')(Line No : 37)
3. do npm test AssignBadgePopup.test.jsx
4. check if all the test cases pass

## Screenshots or videos of changes:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/89441435/acbe5508-3dc8-48bc-9656-2fb939c9264a)


## Note : 
Kindly take a look at the test cases and  suggest if more validations can be added
